docs/codeql/ql-language-reference/modules.rst

```diff
@@ -273,4 +273,69 @@ The ``<module_expression>`` itself can be a module name, a selection, or a quali
 reference. For more information, see ":ref:`name-resolution`."
 
 For information about how import statements are looked up, see "`Module resolution <https://codeql.github.com/docs/ql-language-reference/ql-language-specification/#module-resolution>`__"
-in the QL language specification. 
\ No newline at end of file
+in the QL language specification. 
+
+Built-in modules
+****************
+
+QL defines a `QlBuiltins` module, which is a built-in module that is always in scope.
+Currently, it defines a single parameterized sub-module
+`EquivalenceRelation`, that provides an efficient abstraction for working with
```

Comment:
```suggestion
``EquivalenceRelation``, that provides an efficient abstraction for working with
```

Looking at the preview and at the rest of this file, it seems that identifiers should be in double backticks. This also applies to everything below here.